### DISCUSSION TITLE
Touch Slide Drawer

### DISCUSSION THREAD

**Discussion by fundaxi** [quasarframework/quasar#10210] [General - Components / Directives / etc] [answered]
Hello everyone!

I've searched around but couldn't find anything close to what I want to achieve. 

I'd like to achieve a fixed "tab" on the edge of the screen (left, right, or bottom) that can be expanded similar to the bottom sheet plugin by either clicking this tab or the ability to pull it open via cursor or touch events. I thought the touch pan directive would be the way to go but I'm not certain. I created a very basic codepen example while playing around.

https://codepen.io/kenyon54/pen/WNjMybE

Appreciate any resource links, general pointers, or help with this matter. Thanks!

**Comment by pdanpdan** [comment, marked as answer]
It's not bad - it just needs some fine tuning (make it react on click, add some restrictions on the position).
You can take a look at this: https://github.com/quasarframework/quasar/issues/7541#issuecomment-667349846 (it's for Qv1)

**Reply by fundaxi** [reply]
My searching skills need some improvement! That definitely points me in the right direction. Thanks a bunch!
